main.js

```diff
@@ -1,10 +1,25 @@
 // Vendor
-const {app, BrowserWindow} = require('electron');
+const {app, BrowserWindow, ipcMain} = require('electron');
 const settings = require('electron-settings');
 
 // Constants
 const DEVTOOL_OPTIONS = {mode: 'detach'};
 
+const OVERLAY_SIZE = [
+  {
+    height: 70,
+    width: 460
+  },
+  {
+    height: 60,
+    width: 400
+  },
+  {
+    height: 50,
+    width: 350
+  }
+];
```

Comment:
I'd change the setting to be a "name" instead of an index. The `OVERLAY_SIZE[settings.get('overlay.size')]` would still work and it would be cleaner imo.

```suggestion
const OVERLAY_SIZE = {
  large: {
    height: 70,
    width: 460
  },
  medium: {
    height: 60,
    width: 400
  },
  small: {
    height: 50,
    width: 350
  }
};
```